### PR TITLE
chore: add MANIFEST.in

### PR DESCRIPTION
**Pull request by ebezzi**

### CODE REVIEW

File: cellxgene_schema_cli/MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1 @@
+include requirements.txt
```
Comment (maniarathi) [inline cellxgene_schema_cli/MANIFEST.in:1]: need a newline at the end of the file.

**Review by maniarathi — COMMENTED** [review]
I'm wondering if this is the right place to put it. This is for generating the PyPI release, but all the setup files are in `single-cell-curation/cellxgene_schema_cli/` so would that be a more appropriate place?

Secondly you might also want to include the ontology files located here: `cellxgene_schema_cli/cellxgene_schema/ontology_files/` no?

**Comment by ebezzi** [comment]
> I'm wondering if this is the right place to put it. This is for generating the PyPI release, but all the setup files are in `single-cell-curation/cellxgene_schema_cli/` so would that be a more appropriate place?
> 
> Secondly you might also want to include the ontology files located here: `cellxgene_schema_cli/cellxgene_schema/ontology_files/` no?

Thanks Arathi, good point about the ontology files, they should definitely be bundled as part of the release (unless we move to a download model).

The file is definitely in the `cellxgene_schema_cli/` folder and it is in the same place as `setup.py`, so I believe the location is correct.

**Review by pablo-gar — APPROVED** [review]

File: cellxgene_schema_cli/MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+include cellxgene_schema/ontology_files/*.gz
+include cellxgene_schema/ontology_files/owl_info.yml
+include cellxgene_schema/schema_definitions/*.yaml
```
Comment (maniarathi) [inline cellxgene_schema_cli/MANIFEST.in:4]: Add a newline at the end of the file.

**Pull request merged**